corehq/apps/reports/generic.py

```diff
@@ -857,11 +858,20 @@ def export_table(self):
         3. str(cell)
         """
         headers = self.headers
+        # Strip HTML tags using regex. Regex is much faster than using an HTML
+        # parser, but will strip "<2 && 3>" from a value like "1<2 && 3>2". A
+        # parser will treat each cell like an HTML document, which might be
+        # overkill, but if using regex breaks values then we should use a
+        # parser instead, and take the knock. Assuming we won't have values
+        # with angle brackets, using regex for now.
```

Comment:
I agree with @millerdev here that this would be a much nicer solution. similar to how the sorting is handled you could support something like:
```
{
    "val": 12,
    "format": "<a href='{url}'>{val}</a>",
    "format_vals": { "url": reverse('foo', val), "val": 12 }
}
```
which would be a nice clean separation of rendering and data. that said, agree we can see how this does and whether we want to come back to it under the tech debt umbrella at a later point. (we will need to solve this same problem in UCR so that'd be a good opportunity to redo this and then fold it back in).